docs/2023/Farmstack/Dashboard Builder/index.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/Yaus/Dashboard Builder
```

Comment:
change it to 
slug: /2023/Farmstack/Dashboard Builder

Reply:
Done